Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Transformer.purs

```diff
@@ -87,6 +87,9 @@ checkShowDistance distance = (distance > 0 && distance <= 50000)
 getQuoteList :: Array QuoteAPIEntity -> Array QLI.QuoteListItemState
 getQuoteList quotesEntity = (map (\x -> (getQuote x)) quotesEntity)
 
+truncate :: Number -> Number
+truncate num = (round (num * 10.0)) / 10.0
```

Comment:
add this function in `Helper.Utils` and to make it generic, send the another parameter to decide how many decimal point we need to truncate.